celery/contrib/django/task.py

```diff
@@ -0,0 +1,21 @@
+import functools
+
+from django.db import transaction
+
+from celery.app.task import Task as BaseTask
+
+
+class Task(BaseTask):
```

Comment:
should we use the name as Task or DjangoTask or anything better?

Reply:
I was asklng myself the same question. 

One the one hand, this is namespaced into a `contrib.django` package, so one could make the argument that naming the class `DjangoTask` might feel redundant. 

But on the other hand, this namespace might be glossed over or even not visible in logs/stacktraces. Might also help tooling to have a distinct class name. I'm also already doing some `import Task as DjangoTask` in the tests, so I see that as a signal that a different name might make sense.

Given all that, I'm leaning towards the name `DjangoTask`.